src/main/java/com/joejoe2/chat/service/channel/GroupChannelServiceImpl.java

```diff
@@ -131,7 +131,7 @@ public SseEmitter subscribe(String fromUserId) throws UserDoesNotExist {
   public void subscribe(WebSocketSession session, String fromUserId) throws UserDoesNotExist {
     userRepository
```

Comment:
lines 132 - 134 occur way too often within this class. Perhaps we can extract it into a separate method.